src/tests/utils.js

```diff
@@ -0,0 +1,29 @@
+export async function currentOrderOnPage(sortingType, inventoryPage) {
```

Comment:
Lets move this file out of tests folder. I would rather have it in separate folder like `utils`

Reply:
Updated